external/coreclr/project.json.template

```diff
@@ -3,8 +3,8 @@
     "{TFM}": {
       "dependencies": {
         "Microsoft.NETCore.Platforms": "1.2.0-beta-24721-02",
-        "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24728-02",
-        "Microsoft.NETCore.TestHost": "1.2.0-beta-24728-02",
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24904-03",
```

Comment:
Did you have to manually update this or did the update task correctly update the json.template file? If it didn't auto-update this can you please file an issue to track that work as we want the auto updates to work here.